pyintegration/tests/test_parquet.py

```diff
@@ -0,0 +1,144 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import os
+import shutil
+import unittest
+
+from deephaven2 import empty_table, dtypes, new_table
+from deephaven2.column import InputColumn
+from deephaven2.parquet import write_table, write_tables, read_table, delete_table, ColumnInstruction
+
+from deephaven2.utils import get_workspace_root
+from tests.testbase import BaseTestCase
+
+
+class ParquetTestCase(BaseTestCase):
+    """ Test cases for the deephaven.ParquetTools module (performed locally) """
+    root_dir = ""
+
+    @classmethod
+    def setUpClass(cls):
+        # define a junk table workspace directory
+        cls.root_dir = os.path.join(get_workspace_root(), 'TestParquetTools')
```

Comment:
This looks like the bad old days of the DHE workspace.  Is there any reason you can't just use a python temp dir
https://docs.python.org/3/library/tempfile.html

Reply:
no, but using the default DH workspace also makes sense as we won't potentially spill garbage outside our isolated env. If you feel too strongly about this, I'd be happy to change to use temp.